daemon/api.go

```diff
@@ -989,8 +1037,8 @@ func snapsOp(c *Command, r *http.Request, user *auth.UserState) Response {
 		return BadRequest("unsupported option provided for multi-snap operation")
 	}
 
-	if inst.Action != "refresh" {
-		return BadRequest("only refresh supported for multi-snap operation right now")
+	if inst.Action != "refresh" && inst.Action != "install" && inst.Action != "remove" {
+		return BadRequest("only refresh/install/remove supported for multi-snap operation right now")
```

Comment:
at this point it might make sense to move this to the `default` of the action switch, below.